modules/cli/src/main/scala/scala/cli/commands/shared/SharedOptions.scala

```diff
@@ -511,9 +511,8 @@ final case class SharedOptions(
 
   def inputs(
     args: Seq[String],
-    defaultInputs: () => Option[Inputs] = () => Inputs.default(),
-    isRunWithShebang: Boolean = false
-  ): Either[BuildException, Inputs] =
+    defaultInputs: () => Option[Inputs] = () => Inputs.default()
+  )(implicit programInvokeData: ScalaCliInvokeData): Either[BuildException, Inputs] =
```

Comment:
NIT might be better to use [Scala 3's `using`/`given` syntax](https://docs.scala-lang.org/scala3/book/ca-given-using-clauses.html), rather than the old implicits for passing `ScalaCliInvokeData` around
```suggestion
  )(using programInvokeData: ScalaCliInvokeData): Either[BuildException, Inputs] =
```